### DISCUSSION TITLE
Can I use the Intune Connector in Community

### DISCUSSION THREAD

**Discussion by ghost** [Keyfactor/ejbca-ce#244] [Q&A] [answered]
Hello!

I've been trying to setup the Intune Connector in the community version, but I've hit so many roadblocks and I'm wondering if it's even possible. I'm just getting the error 400 BAD_REQUEST and I'm starting to think it's not allowed on the community version. Is that true?

Thank you!

**Comment by primetomas** [comment, marked as answer]
You probably hit problems following the guide when trying to configure the SCEP alias in RA mode, which is an Enterprise feature. Sorry for not clearly marking the Intune guide as Enterprise, this will be fixes. 
Thanks.

**Reply by primetomas** [reply]
The documentation is now updated. Thanks again for the report.
